shared/domains/chelonia/chelonia.js

```diff
@@ -641,6 +641,21 @@ export default (sbp('sbp/selectors/register', {
     }
     return this.pubsub
   },
+  // This selector is defined primarily for ingesting web push notifications,
+  // although it can be used as a general-purpose API to process events received
+  // from 'external' sources, with 'external' meaning sources not directly
+  // managed by Chelonia (i.e., the WebSocket connection)
```

Comment:
I'd make this a little bit clearer, as right now it sounds like "the WebSocket connection" is given as an example of a source that is "not directly managed by Chelonia", when it is.

E.g. something like:

```
  // This selector is defined primarily for ingesting web push notifications,
  // although it can be used as a general-purpose API to process events received
  // from other external sources that are not managed by Chelonia itself (i.e. sources
  // other than the Chelonia-managed websocket connection and RESTful API).
```